sansa-owl-spark/src/test/scala/net/sansa_stack/owl/spark/rdd/OWLXMLSyntaxOWLAxiomsBuilderTest.scala

```diff
@@ -0,0 +1,648 @@
+package net.sansa_stack.owl.spark.rdd
+
+import com.holdenkarau.spark.testing.SharedSparkContext
+import org.apache.spark.sql.SparkSession
+import org.scalatest.FunSuite
+import org.semanticweb.owlapi.model._
+
+import net.sansa_stack.owl.spark.owl._
+
+class OWLXMLSyntaxOWLAxiomsBuilderTest extends FunSuite with SharedSparkContext {
+
+  lazy val spark = SparkSession.builder().appName(sc.appName).master(sc.master)
+    .config(
+      "spark.kryo.registrator",
+      "net.sansa_stack.owl.spark.dataset.UnmodifiableCollectionKryoRegistrator")
+    .getOrCreate()
+
+  var _rdd: OWLAxiomsRDD = null
+  val syntax = Syntax.OWLXML
+
+  val filePath = this.getClass.getClassLoader.getResource("ont_OWLXML.owl").getPath
+
+  def rdd: OWLAxiomsRDD = {
+    if (_rdd == null) {
+      _rdd = spark.owl(syntax)(filePath)
+      _rdd.cache()
+    }
+    _rdd
+  }
+
+  test("The number of axioms should match") {
+    val expectedNumberOfAxioms = 94
+    assert(rdd.count() == expectedNumberOfAxioms)
+  }
+
+  // //////////// Annotation Properties /////////////////////
+
+
+  test("The number of generated OWLAnnotationAssertionAxiom objects should be correct") {
+   /*
+      <owl:Class rdf:about="http://ex.com/bar#Cls1">
+          <bar:label>Class 1</bar:label>
+          <owl:annotatedSource rdf:resource="http://ex.com/bar#Cls1"/>
+          <owl:annotatedProperty rdf:resource="http://www.w3.org/1999/02/22-rdf-syntax-ns#type"/>
+          <owl:annotatedTarget rdf:resource="http://www.w3.org/2002/07/owl#Class"/>
+          <foo:ann>some annotation</foo:ann>
+      </owl:Class>
+   */
+
+    // The output is
+    // --> AnnotationAssertion(<http://ex.com/foo#ann> <http://ex.com/bar#Cls1> "some annotation"^^xsd:string)
+    // --> AnnotationAssertion(<http://ex.com/bar#label> <http://ex.com/bar#Cls1> "Class 1"^^xsd:string)
```

Comment:
To me those functional syntax comments would be confusing if I read this without knowing where the overall file content stems from. I'd suggest to remove them.

Reply:
ok I will do